.gitignore

```diff
@@ -24,6 +24,10 @@ cachegrind.out*
 *.kdev4
 # Codelite (among others) project files
 *.project
+# ctags index files
+tags
+# GDB initialization scripts
+.gdbinit
```

Comment:
Is there a reason you need to put this script into the RIOT repository? Why not into your home directory?